packages/atomic-angular/scripts/build-lit.mjs

```diff
@@ -0,0 +1,58 @@
+import cem from '@coveo/atomic/custom-elements-manifest' with {type: 'json'};
+import { createWriteStream, readFileSync, writeFileSync } from 'fs';
+
+const isLitDeclaration = (declaration) => declaration?.superclass?.name === 'LitElement';
+
+const atomicAngularModuleFilePath ='projects/atomic-angular/src/lib/stencil-generated/atomic-angular.module.ts';
+const atomicAngularModuleFileContent = readFileSync(atomicAngularModuleFilePath, 'utf-8');
+const atomicAngularComponentFileStream = createWriteStream('projects/atomic-angular/src/lib/stencil-generated/components.ts', {flags: 'a'});
+const litDeclarations = [];
+
+
+const declarationToProxyCmp = (declaration) =>
+`
+@ProxyCmp({
+  inputs: [${declaration.attributes.map(attr => `'${attr.name}'`).join(', ')}]
+})
+@Component({
+  selector: '${declaration.tagName}',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: '<ng-content></ng-content>',
+  // eslint-disable-next-line @angular-eslint/no-inputs-metadata-property
+  inputs: [${declaration.attributes.map(attr => `'${attr.name}'`).join(', ')}]
+})
+export class ${declaration.name} {
+  protected readonly el: HTMLElement;
+  constructor(c: ChangeDetectorRef, el: ElementRef, protected z: NgZone) {
+    c.detach();
+    this.el = el.nativeElement;
+  }
+}
```

Comment:
This is missing events added using **proxyOutputs**. An example below.
 
https://github.com/coveo/ui-kit/blob/4c509698e894d55afb1a1d50b50cec8f0d42d279/packages/atomic-angular/projects/atomic-angular/src/lib/stencil-generated/components.ts#L243

Reply:
Fair! let's do it in another ticket tho https://coveord.atlassian.net/browse/KIT-3802